CoffeeChat/View/PausePearView.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  PausePearView.swift
+//  CoffeeChat
+//
+//  Created by Amy Chin Siu Huang on 10/15/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class PausePearView: UIView {
+
+    // MARK: - Private View Vars
+    private let pauseLabel = UILabel()
+    private let oneWeekButton = UIButton()
+    private let twoWeekButton = UIButton()
+    private let threeWeekButton = UIButton()
+    private let cancelButton = UIButton()
+    
+    // MARK: - Private Data Vars
+    private weak var delegate: PausePearDelegate?
+    
+    init(delegate: PausePearDelegate) {
+        super.init(frame: .zero)
+        self.delegate = delegate
+        setUpViews()
+        setupConstraints()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setUpViews() {
+        self.backgroundColor = .backgroundLightGreen
+        self.layer.cornerRadius = 36
+        
+        pauseLabel.text = "Pause Pear for..."
+        pauseLabel.font = ._20CircularStdBook
+        pauseLabel.textAlignment = .center
+        pauseLabel.textColor = .black
+        addSubview(pauseLabel)
+        
+        oneWeekButton.setTitle("1 Week", for: .normal)
```

Comment:
for these buttons, i think it'd cut down on the amount of repeated code if you had a function that did most of the repeated styling (like corner radius, background color, title color, font, etc) that you could call to style the button, and then set the text. i think there's examples of this elsewhere in pear's codebase, but if it doesn't make sense lmk!